cmd/avalanche.go

```diff
@@ -91,7 +99,7 @@ func main() {
 
 	stop := make(chan struct{})
 	defer close(stop)
-	updateNotify, err := metrics.RunMetrics(*metricCount, *labelCount, *seriesCount, *seriesChangeRate, *maxSeriesCount, *minSeriesCount, *metricLength, *labelLength, *valueInterval, *labelInterval, *metricInterval, *seriesChangeInterval, *seriesOperationMode, *constLabels, stop)
+	updateNotify, err := metrics.RunMetrics(*metricCount, *labelCount, *seriesCount, *seriesChangeRate, *maxSeriesCount, *minSeriesCount, *metricLength, *labelLength, *valueInterval, *labelInterval, *metricInterval, *seriesChangeInterval, *spikeMultiplier, *seriesOperationMode, *constLabels, stop)
```

Comment:
We don't need to do it now, but we should look at breaking `RunMetrics` into separate functions for each of the run modes. The current functions such as this new `handleSpikeMode` could create the other go routines that we create within `RunMetrics` currently.

This would mean we don't have to pass so many parameters around across a few functions.